cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -0,0 +1,86 @@
+from collections import defaultdict
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_list_arg
+
+
+class AddHelpText(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "entries": {
```

Comment:
I'm not sure about "entries" here. It makes sense in a picklist context, but it may not be very clear here. How about `"fields"`?